spec/system/about_page_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+describe "About page", type: :system do
+  fab!(:user)
```

Comment:
```suggestion
  fab!(:current_user) { Fabricate(:user) }
```

minor, but I think it's a good general rule to keep the currently logged in user named `current_user` in system specs, makes it easier to follow